pymtl3/passes/sverilog/translation/behavioral/SVBehavioralTranslatorL1.py

```diff
@@ -66,6 +66,9 @@ def _trim( py_src ):
       for idx, line in enumerate( py_src ):
         if line:
           py_src[ idx ] = line[indent:]
+
+    # TODO get rid of inspect here and use the original lambda source
+    # for those update blocks generated from lambda functions.
     try:
       upblk_py_src = inspect.getsource( upblk )
```

Comment:
also here since you already get the source of the update block from the API, is it possible to remove the usage of inspect here?

Reply:
Yes this inspection here is no longer necessary